parsers/fortran.c

```diff
@@ -110,6 +110,7 @@ typedef enum eKeywordId {
 	KEYWORD_pointer,
 	KEYWORD_precision,
 	KEYWORD_private,
+        KEYWORD_procedure,
```

Comment:
style: spaces instead of a tab.  (this doesn't seem to be in the original patch)